noggin/form/base.py

```diff
@@ -65,3 +66,19 @@ class SubmitButtonField(SubmitField):
 
 def strip(value):
     return value.strip() if value else value
+
+
+class CSVListField(Field):
+    widget = TextInput()
+
+    def _value(self):
+        if self.data:
+            return ','.join(self.data)
+        else:
+            return ''
+
+    def process_formdata(self, values):
+        if values.strip():
```

Comment:
Actually I was wrong, `values` is already a list so it's `values[0]` that should be stripped :-)
And it would be best to do it in the list comprehension below because that list can be empty.

Reply:
and that is why i shouldnt try to code during meetings, lol.

I think i have it sorted now and should cover all the cases, like `[]` and `[" "]` and `[" , , "]`

also added a new test to check such cases